test/chart/AreaChart.perf-test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { measurePerformance } from 'reassure';
+import { Area, AreaChart, Brush, Tooltip, XAxis, YAxis } from '../../src';
+
+type ExampleData = {
+  name: string;
+  uv: number;
+  pv: number;
+  amt: number;
+};
+
+function createDataArray() {
+  const data: ExampleData[] = [];
+  for (let i = 0; i < 1000; i++) {
+    data.push({
+      name: `Page ${i}`,
+      uv: i * 100,
+      pv: i * 200,
+      amt: i * 300,
+    });
+  }
+  return data;
+}
+
+describe('AreaChart', () => {
+  test('Simple test', async () => {
+    const data = createDataArray();
```

Comment:
I wonder whether it would reduce the run duration if we were to create the data only one time when loading the file - as opposed to every time the test is running?

Reply:
I added some measurements around my test like this:

```
describe('AreaChart', () => {
  test('Simple test', async () => {
    console.time('createDataArray');
    const data = createDataArray();
    console.timeEnd('createDataArray');
    console.time('measurePerformance');
    await measurePerformance(
      <AreaChart width={100} height={50} data={data}>
        <Area type="monotone" dataKey="uv" stroke="#ff7300" fill="#ff7300" />
        <Tooltip />
        <XAxis />
        <YAxis />
        <Brush />
      </AreaChart>,
    );
    console.timeEnd('measurePerformance');
  });
});
```

The result is:
- Total run time: 15.115 seconds
- Data generation: 1 millisecond
- `measurePerformance` call: 3643 milliseconds
- "Other" (I suspect jest and reassure and npm): 12 seconds

... so: no, I don't think it's worth it optimizing the `createDataArray` call.

<img width="575" alt="image" src="https://github.com/recharts/recharts/assets/1100170/73336aa5-703d-4b27-9106-aa8c3ceeb628">

<img width="204" alt="image" src="https://github.com/recharts/recharts/assets/1100170/c4d3c924-235c-440c-b920-4e9b1d2cd260">

Reply:
Thank you for looking into it. I was just curious, appreciate your answer.